kloppy/domain/models/common.py

```diff
@@ -151,6 +151,22 @@ def full_name(self):
             return f"{self.first_name} {self.last_name}"
         return f"{self.team.ground}_{self.jersey_no}"
 
+    @property
+    def starting_position(self):
+        if self.position_changes:
+            return self.position_changes[0]["position"]
+
+    def position(self, period: "Period", timestamp: float):
```

Comment:
I've chosen to only support `timedelta` type for `timedelta`. It felt that the simplicity of being able to pass a float instead of a `timedelta` object in the deserializers, didn't outweigh the additional logic required to support both types in the `Player` methods related to handling the position changes.

If you disagree, happy to reconsider!